d8content/metadatadisplay_entity_15.json

```diff
@@ -1,12 +1 @@
-      "langcode": "und",
-      "mimetype": "application/ld+json"
-    }
-  }
-}
+{"data":{"id":"7ed7f7b2-7690-402f-93ea-a9b50a094f30","attributes":{"mimetype":"application\/xml","revision_translation_affected":null,"revision_created":"2024-03-08T16:54:36+00:00","twig":"<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\r\n<OAI-PMH xmlns=\"http:\/\/www.openarchives.org\/OAI\/2.0\/\" \r\n         xmlns:xsi=\"http:\/\/www.w3.org\/2001\/XMLSchema-instance\"\r\n         xsi:schemaLocation=\"http:\/\/www.openarchives.org\/OAI\/2.0\/\r\n         http:\/\/www.openarchives.org\/OAI\/2.0\/OAI-PMH.xsd\">\r\n <responseDate>{{ data_api.request_date }}<\/responseDate>\r\n {%- if data|length > 0 -%}\r\n   {%- set resumptiontoken_arguments = data_api.query %}\r\n   {%-  if data_api.query.page is defined and not empty -%}\r\n     {%- set resumptiontoken_arguments = resumptiontoken_arguments|merge({\"page\":(data_api.query.page+1)})-%}\r\n   {%- else -%}\r\n     {%- set resumptiontoken_arguments = resumptiontoken_arguments|merge({\"page\":2})-%}\r\n   {% endif %}\r\n   {%-\r\n   set resumptiontoken = '?' ~  resumptiontoken_arguments|url_encode\r\n   -%}\r\n {%- else -%}\r\n {# No results so no resumption token #}\r\n  {%- set resumptiontoken = '' -%}\r\n {%- endif -%}\r\n <request {%- for var,value in data_api.query %} {{var}}=\"{{value}}\" {% endfor -%}>{{ resumptiontoken }}<\/request>\r\n {%- if data_api.query.verb == \"ListSets\" -%} \r\n\t\t<error code=\"noSetHierarchy\">This repository does not support sets<\/error>\r\n {%- else -%}\r\n   {%- if data_api.query.verb -%}\r\n <{{ data_api.query.verb }}>\r\n   {%- else -%}\r\n <GetRecord>\r\n   {%- endif -%} \r\n   {%- for XML_doc in data -%}\r\n     {%- if data_api.query.verb == \"ListRecords\" -%}\r\n <record>\r\n     {%- endif -%}\r\n   {{ XML_doc }}\r\n     {%- if data_api.query.verb == \"ListRecords\" -%}  \r\n <\/record>\r\n     {%- endif -%}\r\n   {% endfor %}\r\n {#   {{ data_api|json_encode|raw }} #}\r\n  {#  {{ data_api_context|json_encode|raw }} #}\r\n   {%- if data_api.query.verb -%}\r\n <\/{{ data_api.query.verb }}>\r\n   {%- else -%}\r\n <\/GetRecord>\r\n   {%- endif -%}\r\n {%- endif -%}\r\n<\/OAI-PMH>","name":"OAI-PMH Wrapper","langcode":"und","revision_log_message":null,"drupal_internal__vid":17},"type":"metadatadisplay_entity--metadatadisplay_entity"}}
```

Comment:
@alliomeria heys. This needs to keep the old UUID. Which was 9be8473c-3fb3-42ee-aef2-431b2d4f93bb.